scripts/logstash-run.sh

```diff
@@ -6,7 +6,7 @@ export PATH=$BUILD_DIR/gradle/bin:$PATH
 SERVICE_URL="http://integration:9200"
 
 if [[ "$SECURE_INTEGRATION" == "true" ]]; then
-  SERVICE_URL="https://integration:9200 -k -u admin:admin"
+  SERVICE_URL="https://integration:9200 -k -u admin:myStrongPassword123!"
```

Comment:
You could set this to `admin` to help the build pass.

Reply:
i don't think that is the root-cause of failures but I maybe wrong here.
In the latest CI run, the test run with logstash 7.17.1 with OpenSearch 2.1.0 seem to have passed[1] but the same failed[2] for 1.3.4, when neither of these versions contain admin changes.
[1] - https://github.com/opensearch-project/logstash-output-opensearch/actions/runs/7227042889/job/19693902342?pr=233
[2] - https://github.com/opensearch-project/logstash-output-opensearch/actions/runs/7227042889/job/19693901900?pr=233